SCons/Tool/link.py

```diff
@@ -46,7 +43,9 @@
 
 from SCons.Tool.cxx import iscplusplus
 
-from SCons.Tool import ShLibSonameGenerator
+from SCons.Tool.linkCommon import StringizeLibSymlinks, ShLibSonameGenerator, EmitLibSymlinks, ShLibSymlinkGenerator, \
```

Comment:
IMO these look better one-per-line, Black-style, rather than trying to self-format with line breaks... as in:
```
from SCons.Tool.linkCommon import (
    StringizeLibSymlinks,
    ShLibSonameGenerator,
    EmitLibSymlinks,
    ShLibSymlinkGenerator,
    LdModSymlinkGenerator,
    ShLibPrefixGenerator,
    ShLibSuffixGenerator,
    LdModPrefixGenerator,
    LdModSuffixGenerator,
    LdModSonameGenerator,
)
```